metricbeat/module/windows/perfmon/perfmon_test.go

```diff
@@ -178,7 +177,7 @@ func TestNonExistingCounter(t *testing.T) {
 	}
 	handle, err := NewReader(config)
 	if assert.Error(t, err) {
-		assert.EqualValues(t, pdh.PDH_CSTATUS_NO_COUNTER, errors.Cause(err))
```

Comment:
From what I can tell these errors are never wrapped, and they implement neither the `Cause` or `Unwrap` method.